pkgs/by-name/Xr-Fragments/package.nix

```
@@ -0,0 +1,46 @@
+{
+  stdenv,
+  fetchgit,
+  haxe,
+  nodejs,
+  yarn,
+  makeWrapper,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xrfragment";
+  version = "0.5.2";
+
+  src = fetchgit {
+    url = "https://codeberg.org/coderofsalvation/xrfragment.git";
+    rev = "v0.5.2";
+    hash = "sha256-19zfnV7pVAAdft/KrXX4xbb3EgLw+M5SoEGXcKHR2kg=";
+  };
+
+  buildInputs = [
```

Correct me if I'm wrong, but I believe that these should be `nativeBuildInputs` since they're tools to build the program rather than runtime dependencies necessary for it to run.

```suggestion
  nativeBuildInputs = [
```